test/helper.ts

```diff
@@ -33,3 +35,17 @@ export function parseAndProcess(input: string, fn: (ast: ESTree.Program) => void
   fn(ast);
   return astring(ast).trim();
 }
+
+const defaultSandbox = {
+  console,
+  module: {
+    exports: {}
+  },
+  require
+};
+
+export function virtualModule(code: string, optionsSandbox = {}) {
```

Comment:
I'm with you on this. I implemented it this way to keep the facade in front of runInNewContext light and lean